### ISSUE TITLE
image on the fronpage does have the wrong link (maxplanck-ie)

### ISSUE THREAD

**Issue by bgruening** [deeptools/HiCExplorer#229]
https://hicexplorer.readthedocs.io/en/documentation/index.html

**Comment by joachimwolff** [comment]
For the homepage of the HiCExplorer I changed the image a bit. Please thumbs up if you agree with these changes and or propose changes. 

https://github.com/joachimwolff/website/blob/master/assets/media/hicex2.png

**Comment by gtrichard** [comment]
Nice!

I agree on adding the different logos, although I'm not sure about the order.

MPI and Unifreiburg should be the first I would say no?

Otherwise, the github link should be provided and maybe the direct link to the galaxy version also, below "HiC Explorer".

What do you think?

**Comment by joachimwolff** [comment]
Yes, we can rearrange this. Do you have a SVG version of this image? I only have the a PNG one and the quality is quite bad.

**Comment by gtrichard** [comment]
I asked @vivekbhr

**Comment by fidelram** [comment]
I will send you the link to the original 120 Mb illustrator file. It is uploading at the moment.

**Comment by gtrichard** [comment]
Fidel gave me the link, here is it @joachimwolff 
http://unishare.ie-freiburg.mpg.de:2915/fop/KrbTQHQL/hicexplorer_old_landscape_image.ai

Apparently this is an old version and some names are missing (also the name order maybe?), it needs a double check.
